website/docs/installation/docker-compose.md

```diff
@@ -69,13 +69,17 @@ docker-compose pull
 docker-compose up -d
 ```
 
-The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.
+The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file. Each time you upgrade to a newer version of authentik, you download a new `docker-compose.yml` file, which points to the latest available version. For more information, refer to the **Upgrading** section in the [Release Notes](../releases).
+
+```
```

Comment:
Probably should also remove this block

Reply:
I meant to remove the code block, since it's kinda contradicting us saying that the compose file should be used for upgrades, and then we have a code block saying how to do the opposite of that